README.md

```diff
@@ -27,13 +24,38 @@ compatibility. When possible, upgrade your Conjur version to match the
 when using integrations, choose the latest suite release that matches your Conjur version. For any 
 questions, please contact us on [Discourse](https://discuss.cyberarkcommons.org/c/conjur/5).
 
-### From PyPI
+### Support Services
+
+- Conjur ***TODO find lowest version***
+- DAP ***TODO find lowest version***
+
+### Installation
+
+#### Install the CLI
+
+You can access the latest release of the CLI to install on your machine by navigating to our 
+[release](https://github.com/cyberark/conjur-api-python3/releases) page. For instructions on how to setup and configure the CLI, see our 
+official documentation ***(TODO add link to docs)*** 
+
+If Python is installed on your machine, you can install the client library by pulling from PyPI:
```

Comment:
@sigalsax shouldn't we link to our docs also here?

Reply:
don't think so because nothing more is needed beyond pulling from pypi